PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -47,7 +47,7 @@ def compare_response(step):
         responseKeys = list(response.keys())
         responseKeys.sort()
         for i in range(len(response)):
-            assert str(responseKeys[i]) == str(keys[i]['keys']), "There was an error with the response" 
+            assert str(responseKeys[i]) == str(keys[i]['keys']), "There was an error with the response: \n{}\n{}".format(responseKeys[i],keys[i]['keys']) 
     
     elif apiCall == 'getNeighbors' or apiCall == 'getTips':
         response = responses['getNeighbors'][nodeId] 
```

Comment:
Why is it only `getNeighbors`? What about `getTips`?

Open an issue about it. The best thing to do is have a generic parser fro responses...

Reply:
https://github.com/iotaledger/iri-regression/issues/27